client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/pages/motion-form/components/amendment-create-wizard/amendment-create-wizard.component.html

```diff
@@ -19,7 +19,7 @@ <h2>{{ 'New amendment' | translate }}</h2>
                     <button
                         mat-button
                         [disabled]="contentForm.value.selectedParagraphs.length === 0"
-                        (click)="matStepper.next()"
+                        (click)="matStepper.next(); checkCanSave()"
```

Comment:
Add a new method. Normally we do not want more than one method call within these event listeners.